keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -334,6 +451,64 @@ Disk usage will overall increase by the used memory of the container and the cha
 Checkpoint archive written to disk can optionally be compressed. The current implementation
 does not compress the checkpoint archive on disk.
 
+To avoid running out of disk space an operator has been introduced: <https://github.com/checkpoint-restore/checkpoint-restore-operator>
+
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+During checkpointing each memory page will be written to disk. Disk usage will increase by
+the size of all memory pages in the checkpointed container. Each file in the container that
+has been changed compared to the original version will also be part of the checkpoint.
+Disk usage will overall increase by the used memory of the container and the changed files.
+Checkpoint archive written to disk can optionally be compressed. The current implementation
+does not compress the checkpoint archive on disk.
+
+To avoid running out of disk space an operator has been introduced: <https://github.com/checkpoint-restore/checkpoint-restore-operator>
+
+### Troubleshooting
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+-->
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+The feature does not care if the API server and/or etcd is unavailable.
+
+###### What are other known failure modes?
+
+- The creation of the checkpoint archive can fail.
+  - Detection: See https://kubernetes.io/docs/reference/node/kubelet-checkpoint-api/
```

Comment:
This document is the reference point, I'd expect the functionality be written down here, even if it means copying that from docs. It's the docs that can either point back to KEPs or copy the contents.